scripts/retrieve_databundle_light.py

```diff
@@ -85,6 +85,8 @@
 import re
 from zipfile import ZipFile
 
+import pandas as pd
+import requests
```

Comment:
Requests is not neede in my opinion: the check on whether the bundles are available is done later on.

Reply:
I have used requests library for obtaining the size of data file from the uploaded web server and the final goal was to download the first bundle that occupies less memory space in the computer. We can also not use requests library but adding additinal line (key:value) specifying bundle sizes in bundle_config.yaml file could also be a convenient solution. What do you think?

Reply:
Maybe we could use the size in terms of numbers of countries contained in that databundle.
As a first approximation, we could estimate that, within the subset of bundles that contain all selected countries, the  databundle with the smallest number of countries may be also the smallest.
Note that each "countries" field of the databundles (e.g. "Africa") gets first parsed into the full list of countries, for africa being around 50 countries.
So using the number of countries as a proxy may work, though not being perfect.
The risk with requests is that we need to make many more requests to servers and, although it is an advanced clear approach (cool idea and implementation :DDD), in this case we could use a simpler approach.
What do you think?

Reply:
I agree that this way it will be very close approximation :). I remodify the function to return bundles that contain minimum number of countries but hold the analyzed ones :)

Reply:
Super :D